source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,237 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Aggregation operations process data in your MongoDB collections and return computed results. The **aggregation pipeline**
+is like a factory assembly line. Along the line are assembly stations, called **aggregation stages** . Like an assembly
+station configured with specific tooling to perform its task, each ``stage`` uses **expressions** to perform some
+operation or combination of operations.
+
+Aggregation Does More
+~~~~~~~~~~~~~~~~~~~~~
+
+Using ``find`` operations, you can:
+
+- select *what* documents to return
+- select *what* fields to return
+- sort the results
+
+Using ``aggregation`` operations, you can:
+
+- perform all ``find`` operations
```

Comment:
I think we should take the approach of being objective and list the drawbacks as well such as memory constraints rather than advocating only for aggregation over find.